prymer/api/primer_pair.py

```diff
@@ -79,22 +79,11 @@ class PrimerPair(OligoLike):
     amplicon_tm: float
     penalty: float
     amplicon_sequence: Optional[str] = None
-    _amplicon: Span = field(init=False)
 
-    def __post_init__(self) -> None:
```

Comment:
The problem here is that the validation that the LP/RP are oriented sensibly with respect to each other now won't run until/unless a user accesses the amplicon span.

Reply:
tl;dr The validation logic is still called, and further refactoring would be related to #57

Reply:
Ugh, ok.  So a potential foot-gun to be dealt with as soon as we deal with PrimerLike.